native/src/routes/OffersContainer.tsx

```diff
@@ -37,83 +23,51 @@ type OffersContainerProps = {
 const OffersContainer = ({ navigation, route }: OffersContainerProps): ReactElement => {
   const showSnackbar = useSnackbar()
   const { cityCode, languageCode } = useCityAppContext()
+  const { data, ...response } = useLoadOffers({ cityCode, languageCode })
   const { navigateTo } = useNavigate()
-  const cities = useCities()
-  const { t } = useTranslation('offers')
 
-  const routeInformation = { route: OFFERS_ROUTE, languageCode, cityCode }
-  useSetShareUrl({ navigation, routeInformation, route })
+  const availableLanguages = data?.languages.map(it => it.code)
+  const shareUrl = urlFromRouteInformation({ route: OFFERS_ROUTE, languageCode, cityCode })
+  useHeader({ navigation, route, availableLanguages, data, shareUrl })
 
-  const request = useCallback(async () => {
-    const apiUrl = await determineApiUrl()
-    return createOffersEndpoint(apiUrl).request({
-      city: cityCode,
+  const navigateToOffer = (tile: TileModel) => {
+    const { title, path, isExternalUrl, postData } = tile
+    if (isExternalUrl && postData) {
+      // HTTP POST is neither supported by the InAppBrowser nor by Linking, therefore we have to open it in a webview
```

Comment:
By moving `isExternalUrl && postData` condition to a own variable with a meaningful name, we can get rid of this comment.

Reply:
Do you have a good suggestion for a name?

Reply:
Maybe canOnlyBeHandledByWebview

Reply:
Hmm sorry but I feel like the comment still provides additional information since HTTP Post is the problem here. Will keep it like it was before.